app/src/androidTest/java/com/nervousfish/nervousfish/test/LoginActivitySteps.java

```diff
@@ -56,7 +56,6 @@ public void iHaveALoginActivity() {
         intent.putExtra(ConstantKeywords.SERVICE_LOCATOR,
                 getCurrentActivity().getIntent().getSerializableExtra(ConstantKeywords.SERVICE_LOCATOR));
         getActivity().startActivity(intent);
-        assertTrue(getCurrentActivity() instanceof LoginActivity);
```

Comment:
You don't want this check because the activity isn't set to LoginActivity immediately, that's done asynchronously and often isn't finished yet at that line